tree/treeplayer/inc/ROOT/TDFInterface.hxx

```diff
@@ -1090,6 +1090,10 @@ protected:
                (void)expander; // avoid unused variable warnings for older compilers such as gcc 4.9
             }
             trees[slot]->Fill();
+            if (files[slot]->GetFileBytesWritten() > 96L*1024L*1024L) {
```

Comment:
We just want to avoid gobbling up all the memory if the user has a large snapshot action using something like Pythia event generation. It not really a bug that needs a test. I do plan to make it such that `GetFile()` from `TBufferMerger` can give you something that automatically flushes at a configurable size, but that's for later, we need this simpler control for the release of 6.10.